backend/config.py

```diff
@@ -14,17 +13,24 @@
 
 from pathlib import Path
 
-load_dotenv(find_dotenv("../.env"))
+try:
+    from dotenv import load_dotenv, find_dotenv
+
+    load_dotenv(find_dotenv("../.env"))
+except ImportError:
+    print("dotenv not installed, skipping...")
 
 
 ####################################
 # File Upload
 ####################################
 
+DATA_DIR = Path(os.getenv("DATA_DIR", "./data")).resolve()
 
-UPLOAD_DIR = "./data/uploads"
-Path(UPLOAD_DIR).mkdir(parents=True, exist_ok=True)
+UPLOAD_DIR = f"{DATA_DIR}/uploads"
+UPLOAD_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
Why is Path removed? Won't this break the code?

Reply:
fixed